### DISCUSSION TITLE
[Editor] Is it possible to use editor's panel in a React.js project?

### DISCUSSION THREAD

**Discussion by dsaglam94** [google/model-viewer#4437] [Q&A]
<img width="359" alt="Screenshot 2023-08-27 at 16 07 18" src="https://github.com/google/model-viewer/assets/98871279/e6d29760-4bf7-4e45-be6f-6e1bd76af8ae">

I need to use this tab or something similar in my project. I'm curious if it's possible to implement this feature directly to my project as a package or something. Thanks for the answers 🙏

**Comment by WadeMorrison** [comment]
I'm not sure if there is a package, but you can find the code for it [here](https://github.com/google/model-viewer/blob/d10c010fd66817ad1bdb448c24de3ccc2db31c8e/packages/space-opera/src/components/materials_panel/materials_panel.ts). Hopefully that helps!

**Reply by dsaglam94** [reply]
Yeah, there doesn't seem to be a package for it. Checking the initial code gave me a few ideas. Thanks for the answer!
